lsp/src/main/java/org/aya/lsp/server/AyaService.java

```diff
@@ -63,7 +63,7 @@ public void registerLibrary(@NotNull Path path) {
     var compilerFlags = new CompilerFlags(
       CompilerFlags.Message.EMOJI, false, null,
       libraryManager.modulePath.view());
-
+    libraryManager.loadedFiles.remove(filePath);
```

Comment:
What does this do?

Reply:
Remove last time compiled core. Fix vscode re-highlighting (need to change frontend code, too)